packages/electrode-archetype-react-app-dev/package.json

```diff
@@ -115,6 +115,7 @@
     "sugarss": "^1.0.1",
     "sw-precache": "^5.0.0",
     "sw-toolbox": "^3.4.0",
+    "terser-webpack-plugin-legacy": "^1.2.3",
     "uglify-js": "^3.0.26",
```

Comment:
Do we still need these?

Reply:
looks like its used here https://github.com/electrode-io/electrode/blob/v5.x.x/packages/electrode-archetype-react-app-dev/config/webpack/plugins/sw-precache.js#L22 
but uglifyjs-webpack-plugin can go I think

Reply:
Hi AndyWMT, why `use strict` and `minify` flag are removed

Reply:
@1846689910  correct me but as I understand it node js wraps modules by default so 'use strict' is redundant in this case. 

there is no need for the minify as it is not using uglify in dev